plugins/modules/swim_workflow_manager.py

```diff
@@ -937,62 +937,60 @@ def get_device_uuids(self, site_name, device_family, device_role, device_series_
                     for item_dict in item['response']:
                         site_response_list.append(item_dict)
         else:
-            try:
-                response = self.dnac._exec(
-                    family="site_design",
-                    function='get_site_assigned_network_devices',
-                    op_modifies=True,
-                    params={"site_id": site_id},
-                )
-                self.log("Received API response from 'get_site_assigned_network_devices': {0}".format(str(response)), "DEBUG")
-                response = response.get('response')
+            site_names = site_name + ".*"
+            get_site_names = self.get_site(site_names)
+            self.log(get_site_names)
```

Comment:
self.log("Fetched site names: {0}".format(str(get_site_names)), "DEBUG")

Reply:
fixed the code